jenkins/pipelines/ci/ticdc/dm_ghpr_compatibility_test.groovy

```diff
@@ -0,0 +1,235 @@
+                        }
+                    }
+                }
+
+                dir("go/src/github.com/pingcap/ticdc") {
+                    sh """
+                        cp -R /home/jenkins/agent/git/ticdc/. ./
+                        
+                        echo "build binary with previous version"
+                        git checkout -f ${PRE_COMMIT}
+                        export PATH=$PATH:/nfs/cache/gopath/bin:/usr/local/go/bin
+                        make dm_integration_test_build
+                        mv bin/dm-master.test bin/dm-master.test.previous
+                        mv bin/dm-worker.test bin/dm-worker.test.previous
+
+                        echo "build binary with current version"
+                        git checkout -f ${ghprbActualCommit}
+                        make dm_integration_test_build
+                        mv bin/dm-master.test bin/dm-master.test.current
+                        mv bin/dm-worker.test bin/dm-worker.test.current
+                    """
+                }
+
+                stash includes: "go/src/github.com/pingcap/ticdc/**", name: "ticdc", useDefaultExcludes: false
+
+                def tidb_sha1 = sh(returnStdout: true, script: "curl ${FILE_SERVER_URL}/download/refs/pingcap/tidb/${TIDB_BRANCH}/sha1").trim()
+                sh "curl ${FILE_SERVER_URL}/download/builds/pingcap/tidb/${tidb_sha1}/centos7/tidb-server.tar.gz | tar xz"
+
+
+                // binlogctl
+                sh "curl http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz | tar xz"
+                sh "curl https://download.pingcap.org/tidb-tools-test-linux-amd64.tar.gz | tar xz"
+                sh "mv tidb-tools-test-linux-amd64/bin/sync_diff_inspector bin/"
+                //sh "mv tidb-enterprise-tools-latest-linux-amd64/bin/sync_diff_inspector bin/"
+                sh "mv tidb-enterprise-tools-latest-linux-amd64/bin/mydumper bin/"
+                sh "rm -r tidb-enterprise-tools-latest-linux-amd64 || true"
+
+                // use a new version of gh-ost to overwrite the one in container("golang") (1.0.47 --> 1.1.0)
+                sh "curl -L https://github.com/github/gh-ost/releases/download/v1.1.0/gh-ost-binary-linux-20200828140552.tar.gz | tar xz"
+                sh "mv gh-ost bin/"
+
+                stash includes: "bin/**", name: "binaries"
+            }
+        }
+    }
+
+    stage('Compatibility Tests') {
+        def label = "test-${UUID.randomUUID().toString()}"
+        podTemplate(label: label,
+                nodeSelector: 'role_type=slave',
+                containers: [
+                        containerTemplate(name: 'golang',alwaysPullImage: true, image: "${POD_GO_DOCKER_IMAGE}", ttyEnabled: true,
```

Comment:
```suggestion
                        containerTemplate(name: 'golang',alwaysPullImage: false, image: "${POD_GO_DOCKER_IMAGE}", ttyEnabled: true,
```

Reply:
Needn't set `alwaysPullImage`, if we need to update the image, we should change the pod label